cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -42,102 +44,114 @@
+                                <label class="a-label a-label__heading"
+                                       for="{{ field_id }}">
+                                    Topic
+                                </label>
+                                {{ form.render_with_id(form.topics, field_id) }}
+                            </fieldset>
                         </div>
                     </div>
                 {% endif %}
                 {% if controls.authors %}
                     {% set field_id = 'filter' ~ index ~ '_authors' %}
-                    <div class="form-l_col
-                                form-l_col-1-2">
-                        <div class="form-group">
-                            <label class="form-label-header"
-                                   for="{{ field_id }}">
-                                Author
-                            </label>
-                            {{ form.render_with_id(form.authors, field_id) }}
+                    <div class="content-l_col
+                                content-l_col-1-2">
+                        <div class="o-form_group">
+                            <fieldset class="o-form_fieldset">
+                                <label class="a-label a-label__heading"
+                                       for="{{ field_id }}">
+                                    Author
+                                </label>
+                                {{ form.render_with_id(form.authors, field_id) }}
+                            </fieldset>
                         </div>
                     </div>
                 {% endif %}
                 {% if controls.date_range %}
-                    <div class="form-l_col
-                                form-l_col__flush-bottom
-                                form-l_col-1">
-                        <div class="form-group">
-                            <label class="form-label-header">
-                                Date range
-                            </label>
-                            <div class="form-l">
-                                <div class="form-l_col
-                                            form-l_col-1-2">
-                                    <div class="form-group">
-                                        <label class="form-label-header"
-                                               for="{{ 'filter' ~ index ~ '_from_date' }}">
-                                            From:
-                                        </label>
-                                        {{ form.render_with_id(form.from_date,
-                                           'filter' ~ index ~ '_from_date') }}
+                    <div class="content-l_col
+                                content-l_col-1">
+                        <div class="o-form_group">
+                            <fieldset class="o-form_fieldset">
+                                <label class="a-label a-label__heading">
```

Comment:
I know, but now I'm questioning myself 😝. Go with whichever you think is right, we can change the CF example if `legend` is correct.